hawkular-alerts-engine/pom.xml

```diff
@@ -88,6 +88,12 @@
       <version>${version.com.google.guava}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.reactivex</groupId>
+      <artifactId>rxjava</artifactId>
```

Comment:
> I am going to put this further revision on hold...

Regardless of impl, one thing in the PR that I want to promote is the idea of IncomingDataManager, or basically a component where we can perform various filtering/throttling/massaging of the incoming data in an async way.